src/couch_prometheus/src/couch_prometheus_server.erl

```diff
@@ -177,24 +200,27 @@ get_io_stats() ->
     ].
 
 get_message_queue_stats() ->
-    QLenFun = fun(Name) -> message_queue_len(whereis(Name)) end,
-    Queues = lists:map(QLenFun, registered()),
+    QFun = fun(Name) -> {Name, message_queue_len(whereis(Name))} end,
+    Queues = lists:map(QFun, registered()),
+    QueueLens = lists:map(fun({_, Len}) -> Len end, Queues),
+    QueueLenByLabel = lists:map(fun({Name, Len}) -> {[{queue_name, Name}], Len} end, Queues),
```

Comment:
```
# HELP couchdb_erlang_message_queue_size size of message queue
# TYPE couchdb_erlang_message_queue_size gauge
couchdb_erlang_message_queue_size{queue_name="httpd_sup"} 0
couchdb_erlang_message_queue_size{queue_name="rex"} 0
couchdb_erlang_message_queue_size{queue_name="index_server_1"} 0
couchdb_erlang_message_queue_size{queue_name="folsom_metrics_histogram_ets"} 0
...
```

Would it make sense to sort them, or would that be a waste of cpus resources as nobody will normally look at stats that way anyway?

Reply:
I think it's just wasted cycles to sort, yes